CONTRIBUTING.md

```diff
@@ -30,7 +30,7 @@ here: http://guides.rubygems.org/contributing/
 
 ## Getting Started
 
-    $ rake setup
+    $ bundle install
     $ rake test
```

Comment:
Should we change this instruction to `bundle exec rake test` ?

Reply:
No, since I don't think that's going to work... We want tests to run on a fully unbundled environment. I now realize this is confusing so I should at least add a note about it.

Reply:
What is the benefit of using bundler if we want to run things in a unbundled environment?

Reply:
My original use case is that I wanted to vendor [the `uri` gem](https://github.com/ruby/uri) inside `rubygems`, so I wanted to include [the `automatiek` gem](https://github.com/segiddins/automatiek) to help me with that. But, I needed an unreleased version of automatiek, so I wanted to do something like `gem "automatiek", git: "https://github.com/deivid-rodriguez/automatiek", branch: "my_additional_unreleased_changes"`. Using a gemspec doesn't allow that.